libraries/AP_InertialSensor/AP_InertialSensor_Invensense.cpp

```diff
@@ -618,6 +621,7 @@ bool AP_InertialSensor_Invensense::_accumulate_fast_sampling(uint8_t *samples, u
 
         _accum.gyro += _accum.gyro_filter.apply(g);
         _accum.count++;
+        _seqcnt++;
```

Comment:
Doing it this way just removes the need for assumptions about how many samples are integrated in the accumulate method, and makes the actual sample rate explicit, which is important for determining the Nyquist frequency.